spyql/utils.py

```diff
@@ -32,3 +33,12 @@ def isiterable(x):
     from collections.abc import Iterable
 
     return isinstance(x, Iterable) and x is not Null and not isinstance(x, str)
+
+
+def folder(x):
```

Comment:
```suggestion
def get_folder(x):
```
suggesting using a verb to illustrate what function does